data-serving/scripts/convert-data/move_notes.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import argparse
+import pprint
+import pymongo
+
+
+def database(connection_string, db_name):
+    """Connect to mongodb."""
+    client = pymongo.MongoClient(connection_string)
+    return client[db_name]
+
+
+def move_notes(db, collection, source_id, source, dest, dry_run):
+    """Move the values in the 'source' key to the 'dest' key for cases in the 'collection'
+    collection in the 'db' where their caseReference.source_id matches 'source_id'.
+    If 'remove_source' is set, then additionally remove values from the source key.
```

Comment:
remove_source isn't in params

Reply:
Good catch, it was originally but I realised it isn't needed, we never want to copy the notes into both fields. I'll remove it from the docstring!